### PR TITLE
Add Trino 399 release notes

### LINKED ISSUE #14360: Add Trino 399 release notes

### PR DESCRIPTION
**Pull request by colebow**
## Description

Assemble the release notes for the upcoming Trino 399 release. Fixes #14360 

## Non-technical explanation

N/A

## Release notes

(x) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

## Verification for each commit

Format: commit, committer/maintainer, PR/issue number, ✅ / ❌ rn ✅ / ❌ docs
✅ rn - release note added and verified, or assessed to be not necessary, set to ❌ rn before completion
✅ docs - need for docs assessed and merged, or assessed to be not necessary, set to ❌ docs before completion

## 29 Sep 2022 ✅ 

d81ee4f7931bbde2776a130e4b5c90cdd0028e57 @findepi #14276 ✅ rn ✅ docs
8f44094f3801fa83e6429eb626a1b88b89ed7548 @findepi #14290 ✅ rn ✅ docs
09eacf9f0a2e8eb9b2fb03b2e86f504380d15a74 @findepi #14290 ✅ rn ✅ docs
4c33ba74b92b303af08372f7a97c0570e5ded1b1 @findepi #14290 ✅ rn ✅ docs
4daf674aeb619a52396193a07aa5f64e76329242 @findepi #14290 ✅ rn ✅ docs
0df0c74a0624282a827232fc9e7065b618ad7cf6 @findepi #14290 ✅ rn ✅ docs
39e421352715cc3bf189138449169d300506457a @findepi #14290 ✅ rn ✅ docs
4c27be8922e2ce491eae93499f1cc159574001f9 @findepi #14290 ✅ rn ✅ docs
ca113f50afdae15a6e25c6102a17cc0270c4a155 @findepi #14290 ✅ rn ✅ docs
8c63bb91b6b261010e62dec9fc75a5bb9551a265 @findepi #14290 ✅ rn ✅ docs
da1de5e4af3a59257fdeed6a2e14f3010e7c8e2f @findepi #14290 ✅ rn ✅ docs
255280e5a5386467da7c62d3ca16bb14b46f70c8 @ebyhr #14235 ✅ rn ✅ docs
bdc5ec42c88e4d83f4c146d990048ba7cbd218ff @ebyhr #14342 ✅ rn ✅ docs
dc73950c4e2a164c722918c1a8adada7e122b801 @ebyhr #14326 ✅ rn ✅ docs
d3349750f9904fba330c07ecbd0c772a9308a530 @ebyhr #14378 ✅ rn ✅ docs
6b44cf5920941c9d2698985039b31d34c37536d5 @ebyhr #14352 ✅ rn ✅ docs
c5af70132bf824544242a02df13055fc226a0638 @ebyhr #14366 ✅ rn ✅ docs
3cbe70521c17ed674023279fbd1f7a09c3d511b6 @ebyhr #14366 ✅ rn ✅ docs
ddd9f5129fb06642bec90cbeb1ecc0913ae7d3e6 @ebyhr #14295 ✅ rn ✅ docs
0665a37910ebe5f666c9979bd20117fc3c0793cc @ebyhr #14295 ✅ rn ✅ docs
7319cdfb41a6fdc5a10f29a3a6ce7211999a1250 @ebyhr #14295 ✅ rn ✅ docs
bca1099047b80484bc22cca2340e00fbb56c425d @ebyhr #14295 ✅ rn ✅ docs
41ce825757d577f7ef1b1c2e9fd73875a38cb278 @kasiafi #13602 ✅ rn ✅ docs
8e2ea832f6b62bea8e3b564ea2a31c357cad5ae4 @kasiafi #13602 ✅ rn ✅ docs
398ea9e0b7bbc52874aea0f7fe62b5970ccd4407 @kasiafi #13602 ✅ rn ✅ docs
e5b36361bda58b3045c679ea380cc3cf8683c57c @kasiafi #13602 ✅ rn ✅ docs
92e554439799deec10ee924ad1fefecdde45f853 @kasiafi #13602 ✅ rn ✅ docs
e9f2d98820cfc1e3ccca67dafade1802121c620b @kasiafi #13602 ✅ rn ✅ docs
9c641cb403dfedd0b3658c5bce9e25b6d0d63074 @kasiafi #13602 ✅ rn ✅ docs
c026914d4ed460b75bd6d5849949f0796292e987 @kasiafi #13602 ✅ rn ✅ docs
303d1898ab140768f0d275a0f5b1ac38a8518f5a @kasiafi #13602 ✅ rn ✅ docs
e257357b9703eb74d4d21be49e2a3281db2752d8 @kasiafi #13602 ✅ rn ✅ docs
c899f32437c9a165c95aa137dc3dafdcc43a62ec @kasiafi #13602 ✅ rn ✅ docs
6d9b9ed1ecc5aafad61472ee86395a8a003aa270 @kasiafi #13602 ✅ rn ✅ docs
dfd026457b468da5e257053e03ecc7ea9dc8f890 @kasiafi #13602 ✅ rn ✅ docs
b20fc2e3d5f5cebd5b89d9dba0c1c7529d3a92d0 @kasiafi #13602 ✅ rn ✅ docs
f09633641822f659c1a9e3ddbefa3c5937dffcdc @kasiafi #13602 ✅ rn ✅ docs

## 30 Sep 2022 ✅ 

7aad446d2a3642455976bcbf5a1bafaaf090f589 @sopel39 #14371 ✅ rn ✅ docs
151a2d85101915767899ebf32990622bb9054c28 @sopel39 #14370 ✅ rn ✅ docs
50d1f3f8f508484d4771e69ac54c203a0d273d5b @sopel39 #14370 ✅ rn ✅ docs
b5b514357d01f675a2096bd9358dada4a8648dba @sopel39 #14370 ✅ rn ✅ docs
5c812e31bcd52a49c0c918b79092354d2b808820 @electrum #14384 ✅ rn ✅ docs
87b67b28143608db1ae2507f7968ce9bf625f05e @findepi #14382 ✅ rn ✅ docs
cd286d221d7585669eb7bfb52316ed85c1d1bffc @ebyhr #14363 ✅ rn ✅ docs
332bf74e83667e39834851185f254e699e4153dc @ebyhr #14363 ✅ rn ✅ docs
072fef2a1f6baf7f31e65462899588de29f94fa1 @martint #14409 ✅ rn ✅ docs
389403af94c28e3c6c17ebdca0c95bae2bd170e4 @kokosing #13566 ✅ rn ✅ docs
595a1cfd5cdc70df6857b6c998c656e89677086d @ebyhr #14394 ✅ rn ✅ docs
e4dd2ca8e09eb3967ec506968d2e6f4ed75c28a9 @ebyhr #14356 ✅ rn ✅ docs
0805e46dd1198c4e53343d74e0d41f71a3686ba8 @phd3 #12386 ✅ rn ✅ docs
f84509e875754be66cdd6c92a65282d257f3881f @ebyhr #14411 ✅ rn ✅ docs
ab8337836910568f5c0e2690a391ea8f3a349ece @raunaqmorarka #14309 ✅ rn ✅ docs

## 1 Oct 2022 ✅ 

4078f82073fe01d05cc61e0c6a2082c691511d5e @ebyhr #14416 ✅ rn ✅ docs
171309050423152c650399348445af69c83c4c72 @hashhar #14421 ✅ rn ✅ docs
4a07b2efa3905a49b8c028e62ba3b737d0efc25d @hashhar #14311 ✅ rn ✅ docs
833819e09c361d8fde156e6e76296a90579381e5 @dain #14334 ✅ rn ✅ docs
3c0c5d1b23fe2ca798e468171aee52911d4a04d8 @ebyhr #14399 ✅ rn ✅ docs
41226ec49c2a472e29c87f7e97e12f6f9779f8b0 @ebyhr #14417 ✅ rn ✅ docs
5e5537298fdd63042b77c865e5f8af845baf733f @ebyhr #14413 ✅ rn ✅ docs
97b19ff5e28ef0870aadd535d68d97050565f3c8 @ebyhr #14413 ✅ rn ✅ docs
77f2e12debcae3f6e6665e8b6372da1b171b2660 @ebyhr #14418 ✅ rn ✅ docs

## 2 Oct 2022 ✅ 

c1cb77a095abbc307b2252e4be5553b5ac88f652 @raunaqmorarka #13352 ✅ rn ✅ docs
af1dfa7d464551144e274a3583a963a751b213bd @raunaqmorarka #13352 ✅ rn ✅ docs
666498b208d9bbbb6ea65005fce317050ee7bf0e @raunaqmorarka #13352 ✅ rn ✅ docs
bae640e9797c57d18f1ed60a6ca7d35670c49835 @raunaqmorarka #14350 ✅ rn ✅ docs
bd3dffe7546b9c76b79f8b23a143e6b39f64d517 @ebyhr #14410 ✅ rn ✅ docs
c2232359864b4b163ef7b20e5ecc3654ed82fee9 @ebyhr #14106 ✅ rn ✅ docs
f9cb98aa3ed0858984bfa7858b63a87793c998c9 @ebyhr #14397 ✅ rn ✅ docs

## 3 Oct 2022 ❌ 

a8fbf0f352280e2ea9ed33e40f5713ed8d32e5a7 @ebyhr #14351 ✅ rn ✅ docs
0000565d7095bdfb4de0bd2803591a99b8aaf593 @findepi #14403 ✅ rn ✅ docs
3f14887ab03e3e18136fa511c217359d0de880e0 @ebyhr #14414 ✅ rn ✅ docs
e71e0b5b6d2b715f27d7fc0650386684c6572a31 @ebyhr #14414  ✅ rn ✅ docs
35436920b7dd88098eba39b86557f466fea76995 @ebyhr #14414 ✅ rn ✅ docs
1df86b2ec76e56fbb66bc6a70eb79e18f3aab99f @ebyhr #14414 ✅ rn ✅ docs
678b0198f9cd2693d7758ecf1150af8db4b0c184 @ebyhr #14414 ✅ rn ✅ docs
20dd07b38feb7a6941af850eedd677b2063ede34 @ebyhr #14414 ✅ rn ✅ docs
641cf3ed370b9190563e0eda1d470fea0b510d4c @ebyhr #14414 ✅ rn ✅ docs
a03242904fd360ae82ffc6b606487c9016306bf1 @hashhar #14424 ✅ rn ✅ docs
03594379790d85e6ce18e34a98e622227be8a2b1 @hashhar #14424 ✅ rn ✅ docs
d22bbf5188d804b405a09ebff224e864d977ff12 @hashhar #14424 ✅ rn ✅ docs
bd9c79f044d927de55624b71e85a1c80c8b8f1be @hashhar #14424 ✅ rn ✅ docs
5aa13783025681e3402c5d4fee574a7573c329b5 @hashhar #14436 ✅ rn ✅ docs
303bf604953ec0295b02bc07133e51f0229f6e1d @findepi #14437 ✅ rn ✅ docs
0c5d1939f6fd9a018c08bd67b9e567eac4d14e53 @findepi #14376 ❌ rn ✅ docs
22e5fbd5cebfc7f333491c1b3a345b6bf5a30809 @findepi #14376 ❌ rn ✅ docs
379d811bcc97d853383e4a9c0118b18b153ffca5 @findepi #14376 ❌ rn ✅ docs
91c8c8233e92033237cadcfb60d63d5cc7051cff @kokosing #14153 ✅ rn ✅ docs
cd262b42f2daed20c19ac9db49569266397877d0 @martint #14388 ✅ rn ✅ docs
f7aefc1bf42ad16f634a92630384d7b3881895c1 @electrum #14274 ✅ rn ✅ docs
f1de52a047af36b51420a4c74d8103e940841aeb @electrum #14155 ✅ rn ✅ docs
27fb4166f22a34cbe40f24ec55efc63c77534369 @ebyhr #13942 ✅ rn ✅ docs
ed939550538284e141b868fd68129ff5ed539915 @martint #14415 ✅ rn ✅ docs
9e9ef392322dc4354e2146932072895c0094aa8f @martint #14415 ✅ rn ✅ docs
ab9aecd0867aee242a795b3b4f4901da014893b0 @martint #14415 ✅ rn ✅ docs
391a8c7ea9520095608840df8503c8598d738dcc @raunaqmorarka #14260 ✅ rn ✅ docs
1699f0e3de5334c3e1bb411c3e01bc6c913ce959 @raunaqmorarka #14260 ✅ rn ✅ docs
7b4079a48f448aae864da9eec453f4170b842bfb @martint #14260 ✅ rn ✅ docs
19394a63cd41f7a80b1d023d056b7238f2bc1869 @hashhar #14286 ✅ rn ✅ docs
b39130ae3c1864bd817746d15b51687354081214 @raunaqmorarka #12434 ✅ rn ✅ docs

## 4 Oct 2022 ✅ 

5815511d0983e12ecd9f5969ebcaa78dbe7816df @findepi #13752 ✅ rn ✅ docs
c4c87178bebef3827a390cf6e502270ebe7d5bfe @sopel39 #14408 ✅ rn ✅ docs
9ce7954b929e7e7504b976dd0463e32ecaa8cfd4 @findepi #14412 ✅ rn ✅ docs
48b67241d81c411cea5984166ca513f017dcfe95 @findepi #14335 ✅ rn ✅ docs
113f22d8d770f16d0edca2eeeb05be036101fc80 @arhimondr #14380 ✅ rn ✅ docs
16c906718e6d92cfc6024cc05e4e9b8e6f50fcb1 @dain #14446 ✅ rn ✅ docs
6f1fb1c28ff75f621b26c27739b6ad82b2d79113 @ebyhr #13052 ✅ rn ✅ docs
fa24a13ef743f2ec8646c5cb913fd60eecfb3591 @Praveen2112 #14398 ✅ rn ✅ docs

## 5 Oct 2022

e2566d15fa583adc77f44170dfcc5a168c3e0535 @findepi #14443 ✅ rn ✅ docs
93739c11713dc929699c9d80be20441051e57018 @sopel39 #14212 ✅ rn ✅ docs
d9baf93723baf06c19370cfde89ff3a6aad9295f @sopel39 #14474 ✅ rn ✅ docs
ed009d58b4d4d5d89fbcdbb48ab5fc848288a578 @electrum #14279 ✅ rn ✅ docs
5ab228f537c17711e9509f66bfef1e85736f2f87 @ebyhr #14479 ✅ rn ✅ docs
67843bf089e45acd6e6707cd7ce4837f8c4522a7 @martint #14082 ✅ rn ✅ docs
62133de2dcf663eef31353799a46ae496914cc4d @martint #14082 ✅ rn ✅ docs
e227af748c8d47ecdcd17abf9dde19e689c5235a @martint #14082 ✅ rn ✅ docs
43a3343a7fc72afbd89c2436a44e607a71376422 @martint #14082 ✅ rn ✅ docs
4617d2b2078028b06a813a27e3d4801f2278fe51 @martint #14082 ✅ rn ✅ docs
bb7ad3dc2ec4fac2554b11821d7bc7f1de73fd24 @martint #14082 ✅ rn ✅ docs
33d0b2db5ddea54c63b443fd7b05c02ba8e3c2bc @martint #14082 ✅ rn ✅ docs
6bd6e98fc32159e3dbe1355286da8416d0e27658 @martint #14082 ✅ rn ✅ docs
bf01d9c78f04793dce41e53fcdb302f08d073cea @martint #14082 ✅ rn ✅ docs
199fe72dd72e619bbc6ccd1a2754b72e09ecd596 @martint #14082 ✅ rn ✅ docs
4c15c6cffd67d0bb604b1cc3374ad8793430a3f8 @martint #14082 ✅ rn ✅ docs
0b3a8e663e6885bb820825ef50b333d71c26c983 @martint #14082 ✅ rn ✅ docs
c4095237467311efc2004c6e4817defab658975f @martint #14082 ✅ rn ✅ docs
feae1c78066939df95702b3940560e999b46d277 @martint #14082 ✅ rn ✅ docs
032f1d91328ed73bf8800030fbab2a93de2bf147 @martint #14082 ✅ rn ✅ docs
44312eac3e2dba40e0847160ba5cd4b7db589c8f @martint #14082 ✅ rn ✅ docs
7bb65bf2951a39331abd20ad54b85abe51232256 @martint #14082 ✅ rn ✅ docs
e5b66cf7093adc473f8ac357b2f522b88c2a31dc @martint #14082 ✅ rn ✅ docs
b11ff09993c55890e8db7e4f1f28de5f9b9b232e @martint #14082 ✅ rn ✅ docs
9870de6acbe95cb0e1fb57a33c1ec352d0b91709 @martint #14082 ✅ rn ✅ docs
da0e084226af9ab2d374eaad77c51a9586ec6978 @martint #14082 ✅ rn ✅ docs
dd1e696b0c8d6442c4332a10acb35212193a3ccc @martint #14082 ✅ rn ✅ docs
111405421b195fc55249e9e052b9ed5cce9d2209 @martint #14082 ✅ rn ✅ docs
412258770241feeb9655c9bc7d90deb105b21567 @martint #14082 ✅ rn ✅ docs
5b70291415238ed9f6c2c46c4d005465be0e450e @martint #14082 ✅ rn ✅ docs
ede702182074d323fddec469e6bfa6f33e6908c8 @martint #14082 ✅ rn ✅ docs
8590d247cc97474044426a3d0fa3b0399596cca6 @martint #14082 ✅ rn ✅ docs
9ca222c2fee9c6283a7d50b7db427cde520375c9 @martint #14082 ✅ rn ✅ docs
b3ecae679c62eb71559404f5f4877b23daf7a337 @martint #14082 ✅ rn ✅ docs
dace6fe14a7d58491113d503136f37aa76d33df6 @martint #14082 ✅ rn ✅ docs
8645fbfc67fa37dbaf218b2436fcc04121f39d54 @martint #14082 ✅ rn ✅ docs
57ff92e483897bd119cf20fbb780546e81fc7a46 @electrum #14481 ✅ rn ✅ docs
b0a9488343d18fb014eca6573bdd71417e74263b @ebyhr #14318 ✅ rn ✅ docs
7b6ecd880d6ebb58ed532088ad161beef126bc03 @ebyhr #14480 ✅ rn ✅ docs
c7ea0111079a4bad038ce8dedea6412fbba1fde3 @ebyhr #14480 ✅ rn ✅ docs

## 6 October 2022

7e3e25f0feae5225612e8641fe3fab27b993999f @ebyhr #14473 ✅ rn ✅ docs
0fae006bc2b2eec273be194fa2bcc0f29b040e2e @kasiafi #14469 ✅ rn ✅ docs
b4843822dc188a2e83978bbffa07679a0213561d @kokosing #14221 ✅ rn ✅ docs
a102324c6c9238bb340274c5a52b6bd2cfaab5e1 @sopel39 #14400 ✅ rn ✅ docs
84f773861d0c9bb823804585ec08a1a93ff1d1c7 @findepi #14468 ✅ rn ✅ docs
f0c8ec7367672179fdd4589e08d6d49bdb1cff0e @findepi #14468 ✅ rn ✅ docs
aad961952d399ff615c468c4988d4849a7420973 @findepi #14468 ✅ rn ✅ docs
55c36a0f2d9e5a7b66db4e12b277cfe198d4957d @findepi #14468 ✅ rn ✅ docs
c84fe06a1ddbfc1c07d5c281276c343425de6ca8 @phd3 #13423 ✅ rn ✅ docs
52bbacc22b7f17c5120b8137695fa68d489d5743 @findepi #14458 ✅ rn ✅ docs
22fa9c828d5bbd9e0d7512642f341694bbd83181 @findepi #14458 ✅ rn ✅ docs
842ffc4779030829d3ef2dadb4634b3eeeb2ba13 @findepi #14458 ✅ rn ✅ docs
4c8709f9ad09d1f490a5d2491b4f19fc23a41f1a @findepi #14458 ✅ rn ✅ docs
f1c2c3a48bd5dc047fac15b854718ad8b8782e16 @findepi #14458 ✅ rn ✅ docs
66c1bcd74aa4ebaf90e11b80bce0c72619174f28 @findepi #14458 ✅ rn ✅ docs
89b533ca11bbacceff163f65168051b83f7c49c7 @findepi #14458 ✅ rn ✅ docs
b173ec46a2fe8a9ea6b495fa30ad275105044237 @findepi #14458 ✅ rn ✅ docs
4c64ee0ae3ff2c2f54378e91322af24c06623ba7 @findepi #14458 ✅ rn ✅ docs
418bbc21c9cdf06bf07faf8c26e0d55b02631be9 @findepi #14458 ✅ rn ✅ docs
996850f31846ab1d14290ec34eee83dff5a6b426 @martint #14302 ✅ rn ✅ docs
b7def0df61726ff4b79bfa543d652e9d7df15125 @findepi #14497 ✅ rn ✅ docs
a41381f486e2792741a01c55244d16cd6008a996 @findepi #14497 ✅ rn ✅ docs
60202b2f6654c643bf1be698caf5a563618ab762 @findepi #14497 ✅ rn ✅ docs
1b363fbb03c2c3814c84a3595f86fcc640d98cca @findepi #14462 ✅ rn ✅ docs
1c0512e2afe2177b715aff89dad1dcd255cc2ba5 @findepi #14462 ✅ rn ✅ docs
2b7841773cfeab39bbe7eaaac6eddb67a7272806 @findepi #14462 ✅ rn ✅ docs
f9b75de12802fc5bf9d4811931f85f2d911546d8 @findepi #14462 ✅ rn ✅ docs
4927b3d54b41638a0567fd2b47cb7fd73e1c94ba @findepi #14462 ✅ rn ✅ docs
caab96832fa553a3c81898b778978288aee031a5 @findepi #14462 ✅ rn ✅ docs
a3cc6391fe2a8f415088f6b18edcc0dcfa1dff7c @findepi #14462 ✅ rn ✅ docs
72d962021a897c6b92a9245637720d3b12e757a9 @findepi #14462 ✅ rn ✅ docs
35669f29b2d379fc25f909e531754ba2c2063160 @findepi #14462 ✅ rn ✅ docs
2930362ac274ea1dd9cebb516ec0d63589143ef4 @martint #14488 ✅ rn ✅ docs

### CODE REVIEW

File: docs/src/main/sphinx/release/release-399.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  configuration property to `/trino-accumulo` from `/presto-accumulo`. ({issue}`14326`)
+
+## BigQuery connector
+
+* Add support for writing `array` and `row` columns. ({issue}`14418`)
+
+## ClickHouse connector
+
+* Fix bug where the intended default value of the `domain-compaction-threshold`
+  configuration property was instead a maximum limit. ({issue}`14350`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-399.md:23]: Something is missing here

File: docs/src/main/sphinx/release/release-399.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+## Delta Lake connector
+
+* Improve performance of reading decimal columns from Parquet files. ({issue}`14260`)
+* Allow setting the AWS STS endpoint and region when using a Glue metastore. ({issue}`14412`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-399.md:28]: AWS STS .. might have to spell that out

File: docs/src/main/sphinx/release/release-399.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+## Hive connector
+
+* Add `max-partition-drops-per-query` configuration property to limit the number
+  of partition drops. ({issue}`12386`)
+* Add `hive.s3.region` configuration property to force S3 to connect to a
+  specific region. ({issue}`14398`)
+* Improve performance of reading decimal columns from Parquet files. ({issue}`14260`)
+* Reduce memory usage of the coordinator. ({issue}`14408`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-399.md:37]: Why is this in the Hive connector section? Also  .. memory usage on the coordinator?

File: docs/src/main/sphinx/release/release-399.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+* Reduce query memory usage during inserts to S3. ({issue}`14212`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-399.md:38]: Is this only Hive .. or also others?

File: docs/src/main/sphinx/release/release-399.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+## MongoDB connector
+
+* Support predicate pushdown on `date`, `time(3)`, `timestamp(3)` and
+  `timestamp(3) with time zone` columns. ({issue}`5658`)
+
+## Phoenix connector
+
+* Fix bug where the intended default value of the `domain-compaction-threshold`
+  configuration property was instead a maximum limit. ({issue}`14350`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-399.md:64]: something missing

**Review by mosabua — CHANGES_REQUESTED** [review]

File: docs/src/main/sphinx/release/release-399.md (hunk 3 above)
Comment (colebow) [reply docs/src/main/sphinx/release/release-399.md:37]: It only impacts Hive, it has to do with caching Hadoop LocatedFileStatus objects.

**Review by mosabua — APPROVED** [review]
Looks good now.

**Review by mosabua — APPROVED** [review]
Looks good.

**Pull request merged**